src/bootlint.js

```diff
@@ -937,12 +959,7 @@ var LocationIndex = _location.LocationIndex;
             var elem = $(this);
             var urlAttr = (tagNameOf(this) === 'LINK') ? 'href' : 'src';
             var pathSegments = parseUrl(elem.attr(urlAttr)).pathname.split('/');
-            var matches = pathSegments.map(function (segment) {
-                var match = segment.match(/^\d+\.\d+\.\d+$/);
-                return match ? match[0] : null;
-            }).filter(function (match) {
-                return match !== null;
-            });
+            var matches = getSemverFromURI(pathSegments);
             if (!matches.length) {
                 return;
             }
```

Comment:
The "get the last match" logic ending on the next line should be pushed into getSemverFromURL